README.md

```diff
@@ -16,15 +16,53 @@ flux-accounting requires the following packages to build:
 
 ### Install Instructions
 
-You can install the dependencies required by flux-accounting (located in **requirements.txt**) with the following command:
+You can install the dependencies required by flux-accounting as well as the package itself to be recognized by Flux's command driver `flux(1)` with `make`.
+
+1. Set your **FLUX_INSTALL_PREFIX** environment variable to point to flux-core's installation target:
+
+```
+$ export $FLUX_INSTALL_PREFIX = ~/path/to/flux-core/install/
+```
+
+2. After building flux-core, you'll need to set **PYTHONPATH** to point towards flux-core's Python site-packages as well as the **PATH** to point to Flux's executable:
+
+```
+$ export PYTHONPATH = ~/path/to/flux-core/install/lib/python3.7/site-packages/
```

Comment:
I don't think that exporting `PYTHONPATH` is necessary as long as the `flux` command is used rather than running `flux-account.py` directly.